full/coupler_main.F90

```diff
@@ -331,270 +331,27 @@
-  use atmos_model_mod,         only: atmos_model_restart
-  use atmos_model_mod,         only: update_atmos_model_radiation
-  use atmos_model_mod,         only: update_atmos_model_state
-
-  use land_model_mod,          only: land_model_init, land_model_end
-  use land_model_mod,          only: land_data_type, atmos_land_boundary_type
-  use land_model_mod,          only: update_land_model_fast, update_land_model_slow
-  use land_model_mod,          only: atm_lnd_bnd_type_chksum
-  use land_model_mod,          only: land_data_type_chksum
-  use land_model_mod,          only: land_model_restart
-
-  use ice_model_mod,           only: ice_model_init, share_ice_domains, ice_model_end, ice_model_restart
-  use ice_model_mod,           only: update_ice_model_fast, set_ice_surface_fields
-  use ice_model_mod,           only: ice_data_type, land_ice_boundary_type
-  use ice_model_mod,           only: ocean_ice_boundary_type, atmos_ice_boundary_type
-  use ice_model_mod,           only: ice_data_type_chksum, ocn_ice_bnd_type_chksum
-  use ice_model_mod,           only: atm_ice_bnd_type_chksum, lnd_ice_bnd_type_chksum
-  use ice_model_mod,           only: unpack_ocean_ice_boundary, exchange_slow_to_fast_ice
-  use ice_model_mod,           only: ice_model_fast_cleanup, unpack_land_ice_boundary
-  use ice_model_mod,           only: exchange_fast_to_slow_ice, update_ice_model_slow
-
-  use ocean_model_mod,         only: update_ocean_model, ocean_model_init,  ocean_model_end
-  use ocean_model_mod,         only: ocean_public_type, ocean_state_type, ice_ocean_boundary_type
-  use ocean_model_mod,         only: ocean_model_restart
-  use ocean_model_mod,         only: ocean_public_type_chksum, ice_ocn_bnd_type_chksum
-
-  use combined_ice_ocean_driver, only: update_slow_ice_and_ocean, ice_ocean_driver_type
-  use combined_ice_ocean_driver, only: ice_ocean_driver_init, ice_ocean_driver_end
-!
-! flux_ calls translate information between model grids - see flux_exchange.f90
-!
-
-  use flux_exchange_mod,       only: flux_exchange_init, gas_exchange_init, sfc_boundary_layer
-  use flux_exchange_mod,       only: generate_sfc_xgrid, send_ice_mask_sic
-  use flux_exchange_mod,       only: flux_down_from_atmos, flux_up_to_atmos
-  use flux_exchange_mod,       only: flux_land_to_ice, flux_ice_to_ocean, flux_ocean_to_ice
-  use flux_exchange_mod,       only: flux_ice_to_ocean_finish, flux_ocean_to_ice_finish
-  use flux_exchange_mod,       only: flux_check_stocks, flux_init_stocks
-  use flux_exchange_mod,       only: flux_ocean_from_ice_stocks, flux_ice_to_ocean_stocks
-  use flux_exchange_mod,       only: flux_atmos_to_ocean, flux_ex_arrays_dealloc
-
-  use atmos_tracer_driver_mod, only: atmos_tracer_driver_gather_data
-
-  use iso_fortran_env
-
-  implicit none
-
-!-----------------------------------------------------------------------
-
-  character(len=128) :: version = '$Id$'
-  character(len=128) :: tag = '$Name$'
```

Comment:
These should probably be added back in.  There may be some models that use this.

Reply:
Wondering if they wanted to use version and tag, they could use full_coupler_mod?

Reply:
These are artifacts from our days of using CVS. We had a different method for GitHub, but trying to insert things was shell dependent and not reliable.

Reply:
Sounds like it is safe to remove then.  I will remove it.

Reply:
Spoke too soon, it is used in coupler_init.

Reply:
Look at other files that use version and search back in their history to see how things have changed.  For example here's mpp.F90 from [19 years ago](https://github.com/NOAA-GFDL/FMS/blame/ac1a7aacb7699f4ed8d0544b8127d8782ad8315a/mpp/mpp.F90#L1246-L1249) and [9 years ago](https://github.com/NOAA-GFDL/FMS/blame/e77e353f03b75661733250ce78c5777930230ba3/mpp/mpp.F90#L1246-L1251) when version and tag where updated/removed.

Reply:
I'll do this in a separate PR